test/javascripts/acceptance/chat-test.js

```diff
@@ -225,6 +226,8 @@ acceptance("Discourse Chat - without unread", function (needs) {
     Object.defineProperty(this, "appEvents", {
       get: () => this.container.lookup("service:appEvents"),
     });
+    Session.current().highlightJsPath =
+      "/assets/highlightjs/highlight-test-bundle.min.js";
```

Comment:
I guess this is needed to get highlighting working in tests? Is this related to the usage of `import highlightSyntax from "discourse/lib/highlight-syntax";` in the implementation?

Reply:
Yeah I just copied what another test in core was doing with it, `highlight-syntax` loads this library